bootstrap-salt.sh

```diff
@@ -4538,6 +4542,13 @@ install_amazon_linux_ami_git_deps() {
         yum -y install ca-certificates || return 1
     fi
 
+    PIP_EXE='pip'
+    if [ "$(echo "$GIT_REV" | egrep -o '2[0-9]{3}')" -ge 2016 ]; then
+        [ $(which pip2.7) ] || /usr/bin/easy_install-2.7 pip || echoerror "Could not install pip2.7"
```

Comment:
Yes thank you I made an assumption that echoerror would return 1 for me without actually looking. Thanks for that catch :) And seriously thanks for all your other suggestions. I really appreciate it :)